commit f7132863450d28413a83f8f36c6cf9fb2e2f7c4b (refs/changes/09/5309/5)
Author: Rui Loura <rui@joyent.com>
Date:   2019-01-04T16:22:24+00:00 (9 months ago)
    
    TRITON-721 joyent-schemas should support RAN sysinfo keys
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/lib/cnapi/server.js b/lib/cnapi/server.js
index 088c2fc..21770f2 100644
--- a/lib/cnapi/server.js
+++ b/lib/cnapi/server.js
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Joyent, Inc.  All rights reserved.
+ * Copyright 2019 Joyent, Inc.  All rights reserved.
  */
 
 var schema_util = require('../util.js');
@@ -204,6 +204,8 @@ var s_sysinfo = {
 		'CPU Type': base.str.nonempty.req,
 		'CPU Virtualization': base.str.req,
 		'CPU Physical Cores': base.intg.pos.req,
+		'Admin NIC Tag': base.str.opt, // OS-7075 (20180719)
+		'Admin IP': base.str.v4addr.opt, // TRITON-841 (20181206)
 		'UUID': base.str.uuid.req,
 		'Hostname': base.str.hostname.opt,
 		'HVM API': base.bool.opt,
diff --git a/lib/dapi/server.js b/lib/dapi/server.js
index 3ee2db1..c9e24b2 100644
--- a/lib/dapi/server.js
+++ b/lib/dapi/server.js
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Joyent, Inc.  All rights reserved.
+ * Copyright 2019 Joyent, Inc.  All rights reserved.
  */
 
 var schema_util = require('../util.js');
@@ -50,6 +50,8 @@ var s_sysinfo = {
 		'Bhyve Capable': base.bool.opt,
 		'Bhyve Max Vcpus': base.intg.nonneg.opt,
 		'CPU Total Cores': base.intg.pos.req,
+		'Admin NIC Tag': base.str.opt, // OS-7075 (20180719)
+		'Admin IP': base.str.v4addr.opt, // TRITON-841 (20181206)
 		'HVM API': base.bool.opt,
 		'Network Interfaces': s_si_nics_r,
 		'Live Image': base.str.nonempty.req,
